lib/sentry/rescue_evss_errors.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Sentry
+  module RescueEVSSErrors
+    include SentryLogging
+
+    private
+
+    def rescue_evss_errors(keys)
+      yield
+    rescue EVSS::ErrorMiddleware::EVSSError => e
+      if e.message&.match keys
```

Comment:
At this point, `e.message` is the original `messages` array, but has been converted to string, hence the regex match for the key value.